### PR TITLE
Counter implementation

### PR DESCRIPTION
**Pull request by JuanMaGiraldo**
This PR contains the implementation of the first Live-View video.

### CODE REVIEW

File: lib/counter_web/live/counter_live.ex [hunk 1]
```diff
@@ -0,0 +1,28 @@
+defmodule CounterWeb.CounterLive do
+  use Phoenix.LiveView
+
+  def mount(_session, _, socket) do
```
Comment (juanhurtado10) [inline lib/counter_web/live/counter_live.ex:4]: @JuanMaGiraldo please explain how a mount function works

File: lib/counter_web/live/counter_live.ex [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    socket = assign(socket, :count, 0)
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~L"""
```
Comment (juanhurtado10) [inline lib/counter_web/live/counter_live.ex:10]: @JuanMaGiraldo what this sigil means?

File: lib/counter_web/live/counter_live.ex [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <h1>Count <%= @count %> </h1>
+    <button phx-click="increment">+</button>
+    <button phx-click="decrement">-</button>
+    """
+  end
+
+  def handle_event("increment", _value, socket) do
```
Comment (juanhurtado10) [inline lib/counter_web/live/counter_live.ex:17]: @JuanMaGiraldo what a handle_event does?

**Review by juanhurtado10 — APPROVED** [review]
👏  well job done!

File: lib/counter_web/live/counter_live.ex (hunk 3 above)
Comment (JuanMaGiraldo) [reply lib/counter_web/live/counter_live.ex:17]: It is an implementation of a callback, in Liveview, all bindings are handled with it.

File: lib/counter_web/live/counter_live.ex (hunk 1 above)
Comment (JuanMaGiraldo) [reply lib/counter_web/live/counter_live.ex:4]: `mount/3` is a callback that is invoked when LiveView is first rendered, it's like a constructor for the view.
`mount/3` is invoked with the current params, session and the socket, in this case, I have wrongly positioned the section parameter.

File: lib/counter_web/live/counter_live.ex (hunk 2 above)
Comment (JuanMaGiraldo) [reply lib/counter_web/live/counter_live.ex:10]: It means that html code will be declared inside the quotes.

**Pull request merged**